src/Core_Command.php

```diff
@@ -586,6 +589,10 @@ function wp_new_blog_notification() {
 			'admin_password' => '',
 		];
 
+		if ( \WP_CLI\Utils\wp_version_compare( '4.0', '>=' ) ) {
```

Comment:
`WP_CLI\Utils` was already imported at the top.
```suggestion
		if ( Utils\wp_version_compare( '4.0', '>=' ) ) {
```